drivers/gles2/shader_compiler_gles2.cpp

```diff
@@ -361,6 +361,23 @@ String ShaderCompilerGLES2::_dump_node_code(SL::Node *p_node, int p_level, Gener
 				fragment_global += final_code;
 			}
 
+			// constants
+
+			for (Map<StringName, SL::ShaderNode::Constant>::Element *E = snode->constants.front(); E; E = E->next()) {
+				String gcode;
+				gcode += "const ";
+				gcode += _prestr(E->get().precision);
+				gcode += _typestr(E->get().type);
+				gcode += " " + _mkid(E->key());
+				if (E->get().initializer != NULL) {
```

Comment:
As constants always have to be initilized this if statement is now kinda unnecessary. Also in GLES3.

Reply:
Fixed